Documentation/Storage-Configuration/Advanced/ceph-mon-health.md

```diff
@@ -115,3 +115,9 @@ $ ceph -s
     osd: 3 osds: 3 up (since 10m), 3 in (since 10m)
 [...]
 ```
+
+## Automatic Monitor Failover
+
+Rook will automatically failover the mons when the following settings are updated in the CephCluster CR:
+- `spec.network.hostNetwork`: When enabled or disabled, rook fails over all the monitors to either enable or disable host networking.
```

Comment:
In the cluster.yaml please update `hostNetwork` spec

Reply:
I don't see the `hostNetwork` settings in the cluster.yaml file.  
I can only see ` # enable host networking` comment. Maybe the setting got removed by mistake.

Reply:
The preferred setting now is `network.provider: host`. The setting `network.hostNetwork: true` is an older way that is still supported for backward compatibility, but we don't need to document it.